Cargo.toml

```diff
@@ -42,7 +42,7 @@ repository.workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
 bevy = { workspace = true }
-vello = { git = "https://github.com/linebender/vello", rev = "28914d66f675efecf03c53261a0fb1b8ecce4fe9" }
+vello = { git = "https://github.com/linebender/vello", rev = "f164ffc01710a996ee6e3c0e0783635b3aaef209" }
```

Comment:
Side note, Vello is working on a 0.1 release right now. We want to revisit this in a couple weeks at most and pin to versions instead of git refs. Then we could publish bevy-vello thereafter.